src/core/order_manager.py

```diff
@@ -0,0 +1,35 @@
+# order manager mechanism
+
+from queue import Queue
+
+
+class OrderManager:
+
+    def __init__(self):
+
+        self._order_storage = {}
+        self.order_id_count = 1
+
+    def init_order_storage(self, order_status):
+
+        for status in order_status:
+
```

Comment:
You can remove empty line here

Reply:
Remove unnecessary empty lines in this file

Reply:
done!